### DISCUSSION TITLE
Customize the pxe boot menu

### DISCUSSION THREAD

**Discussion by marbaa** [cobbler/cobbler#3121] [Version specific questions]
Hi,

Cobbler 3.2.2

I'm playing little bit with customization pxe boot menu. I would like to change the default black/gray color to something else + custom backround image.
![image](https://user-images.githubusercontent.com/40859847/170842006-2d177da9-812e-4e1b-956b-e81478847e0c.png)

I'm editing files in ```/etc/cobbler/boot_loader_conf```. By try and error I found out which files needs to be modified. For example to change the font color and selection bar I've added to the ```grubsystem.template``` line ```set menu_color_highlight=cyan/black```
![image](https://user-images.githubusercontent.com/40859847/170842090-410ec79e-8564-4acc-ad2c-fc8e1885ff2b.png)

However I can't load custom background image. I've tried those lines in many files in ```/etc/cobbler/boot_loader_conf```:
```
insmod all_video
insmod gfxterm
insmod gfxterm_menu
insmod gfxmenu
insmod gfxterm_background
insmod png
terminal_output gfxterm
background_image -m stretch /logo.png
```

put the image ```logo.png``` with 640x480 into ```/var/lib/tftpboot```. But still black background.
Which file to edit to load custom background image?

**Comment by SchoolGuy** [comment]
I believe you are the first to attempt this (at least who documented what he tried). I have never tried this myself, so I am not of much help here sadly this time. If you succeed at some point, I would love if you document it in here. :)
